### PR TITLE
Fix dead link

### PR DESCRIPTION
**Pull request by keisuke-umezawa**

### CODE REVIEW

File: docs/source/glance.rst [hunk 1]
```diff
@@ -17,7 +17,7 @@ Chainer is a rapidly growing neural network platform. The strengths of Chainer a
 Mushrooms -- tasty or deathly?
 ------------------------------
 
-Let's take a look at a basic program of Chainer to see how it works. For a dataset, we'll work with `Kaggle's edible vs. poisonous mushroom dataset <https://www.kaggle.com/uciml/mushroom-classification>`_, which has over 8,000 examples of mushrooms, labelled by 22 categories including odor, cap color, habitat, etc., in a `mushrooms.csv file <https://raw.githubusercontent.com/chainer/chainer/master/docs/source/mushrooms.csv>`_.
+Let's take a look at a basic program of Chainer to see how it works. For a dataset, we'll work with `Kaggle's edible vs. poisonous mushroom dataset <https://www.kaggle.com/uciml/mushroom-classification>`_, which has over 8,000 examples of mushrooms, labelled by 22 categories including odor, cap color, habitat, etc., in a `mushrooms.csv file <https://www.kaggle.com/uciml/mushroom-classification#mushrooms.csv>`_.
```
Comment (mitmul) [inline docs/source/glance.rst]: I think the file has been moved to https://github.com/chainer/chainer/raw/master/examples/glance/mushrooms.csv

**Comment by keisuke-umezawa** [comment]
@mitmul I fixed as you said!

**Comment by toslunar** [comment]
LGTM.

#5316 caused the issue.  Sorry.

**Comment by mitmul** [comment]
Jenkins, test this please

**Comment by mitmul** [comment]
LGTM

**Pull request merged**